testsuite/Gemfile

```
@@ -17,7 +17,7 @@ gem 'minitest'
 gem 'websocket'
 gem 'websocket-driver'
 gem 'syntax'
-gem 'parallel_tests'
+gem 'parallel_tests', '3.1.0'
```

```suggestion
gem 'parallel_tests', '3.8.1'
```
According to https://github.com/grosser/parallel_tests/issues/872 we should use version 3.8.1 as a quick fix for now.

---

Yes. But according to our env, 3.8.1 will cause an issue in cascade now, with another dependency requiring ruby 2.7

I prefer to just be in the safe side for once 😁

---

Oh, I did not know that. So then `3.1.0` it is.